lib/configs/vue.ts

```diff
@@ -139,5 +140,16 @@ export function vue(options: ConfigOptions): Linter.Config[] {
 			},
 			name: 'nextcloud/vue/stylistic-rules',
 		},
+
+		{
+			files: GLOB_FILES_VUE,
```

Comment:
Maybe include all files. We also have functions, that can be used in `.js/.ts` files

Reply:
So that would be `[...GLOB_FILES_VUE, ...GLOB_FILES_JAVASCRIPT, ...GLOB_FILES_TYPESCRIPT, ...GLOB_FILES_TESTING]`? Shall we introduce GLOB_* constant for this?